pylint/testutils.py

```diff
@@ -627,9 +628,15 @@ def _check_output_text(self, expected_messages, expected_lines, received_lines):
         expected_lines = self._split_lines(expected_messages, expected_lines)[0]
         for exp, rec in itertools.zip_longest(expected_lines, received_lines):
             assert exp == rec, (
-                "Expected test lines did not match for test: {}."
+                "Wrong output for '{_file}.txt':"
+                "{_linesep}You can update the expected output automatically with: '"
+                'python tests/test_functional.py {} -k "test_functional[{_file}]"\'{_linesep:s}'
                 "{_linesep:s}Expected : {}"
                 "{_linesep:s}Received : {}".format(
-                    self._test_file.base, exp, rec, _linesep=os.linesep
+                    UPDATE_OPTION,
+                    exp,
+                    rec,
+                    _linesep=os.linesep,
+                    _file=self._test_file.base,
```

Comment:
I know that it is me who introduced the mix between keyword and positional arguments in the call to `format` but now that there are more arguments i think it could be more readable to use only keywords. What do you think about it?

Reply:
Moreover, i just read the comment you made on another PR concerning the use of `os.linesep`. I think it should be replaced by `\n` here. Would you mind doing it please?

Reply:
No problem, I'm also going to modify it a bit in #3918  later on.